include/common_package/node_names.hpp

```diff
@@ -0,0 +1,34 @@
+//
+// Created by Johan <job8197@thi.de> on 17.05.2024.
+//
+
+#ifndef THI_DRONE_WS_NODE_NAMES_HPP
+#define THI_DRONE_WS_NODE_NAMES_HPP
+
+#include <set>
+
+namespace common_lib::node_names {
+
+constexpr char const *const MISSION_CONTROL =
+    "mission_control"; /**< Node name for the mission control node */
+
+constexpr char const *const WAYPOINT =
+    "waypoint"; /**< Node name for the navigate to waypoint node */
+
+constexpr char const *const QRCODE_DETECTION =
+    "qrcode_detection"; /**< Node name for the qr code detection node */
+
+constexpr char const *const FCC_BRIDGE =
+    "fcc_bridge"; /**< Node name for the FCC bridge node */
+
+constexpr char const *const TELEMETRY =
+    "telemetry"; /**< Node name for the telemetry node */
+
+const std::set<std::string> VALID_CONTROL_NODE_NAMES{
```

Comment:
Why do we need this?

Reply:
I would like to check if the node being is activated is allowed to do so.
I could put this set into the fcc_bridge but I opted to keep it where the node names are actually defined, so that a change to them can directly be reflected in the valid nodes.

Reply:
Ok, fine by me